src/CommandingOfficers/AW4/NRA/Waylon.java

```diff
@@ -0,0 +1,83 @@
+package CommandingOfficers.AW4.NRA;
+
+import java.util.ArrayList;
+
+import CommandingOfficers.Commander;
+import CommandingOfficers.CommanderInfo;
+import CommandingOfficers.DeployableCommander;
+import CommandingOfficers.AW4.RuinedCommander;
+import Engine.GameScenario;
+import Engine.UnitMods.UnitModifier;
+import Engine.UnitMods.UnitDefenseDoRModifier;
+import Engine.UnitMods.UnitTypeFilter;
+import Terrain.MapMaster;
+import UI.UIUtils;
+import Units.UnitModel;
+
+public class Waylon extends RuinedCommander
+{
+  private static final long serialVersionUID = 1L;
+
+  private static final CommanderInfo coInfo = new instantiator();
+  private static class instantiator extends CommanderInfo
+  {
+    private static final long serialVersionUID = 1L;
+    public instantiator()
+    {
+      super("Waylon", UIUtils.SourceGames.AW4, UIUtils.NW);
+      infoPages.add(new InfoPage(
+          "A member of the Rubinelle Army who seeks only to fulfill his own pleasures and desires.\n"));
+      infoPages.add(new InfoPage(
+          "Base Zone: "+RADIUS+"\n"
+          + "Zone Boost: Air units +"+POWER+"/"+DEFENSE+".\n"));
+      infoPages.add(new InfoPage(new Wingman(null),
+          "Boost air unit defense by 270 (410 + veterancy, total).\n"));
```

Comment:
410 "with" veterancy?

Reply:
410 minimum, up to 430 with veterancy's +20.